diff --git a/wine-tkg-git/customization.cfg b/wine-tkg-git/customization.cfg
index b48b74a..a2d088b 100644
--- a/wine-tkg-git/customization.cfg
+++ b/wine-tkg-git/customization.cfg
@@ -37,7 +37,7 @@ _GENERATE_DEBIAN_PACKAGE="false"
 # Custom presets : "fo4", "legacy", "mwo", "sims2", "oldnvidia-sims3", "protonified", "unity".
 # Valve presets :  "valve" (builds against current valve proton tree), "valve-exp" (builds against current valve proton-experimental tree), "valve-exp-bleeding" (builds against current valve proton-experimental-bleeding-edge tree)
 # Clean presets (untouched) : "mainline", "staging".
-_LOCAL_PRESET=""
+_LOCAL_PRESET="none"
 
 # Add GloriousEggroll game patches and hotfixes when using a Valve profile with staging enabled
 # ! This will only affect Valve + staging trees !
@@ -121,7 +121,7 @@ _assettocorsa_hudperf_fix="false"
 
 # Fixes for Mortal Kombat 11 - Requires staging, _proton_fs_hack="true", native mfplat (win7) or staging mfplat support and a different GPU driver than RADV
 # On Wine 5.2 (up to b1c748c8) and lower, it needs to be toogled on with the WINE_LOW_USER_SPACE_LIMIT=1 envvar
-_mk11_fix="true"
+_mk11_fix="false"
 
 # Workaround for Final Fantasy XIV Launcher 404 error - Thanks @varris1 ! - Fixed by d535df42f665a097ec721b10fb49d7b18f899be9 (4.10)
 # Found to also enable the new launcher (that came with the 5.1 update) to work *with issues*
@@ -167,14 +167,14 @@ _win10_default="false"
 
 # Other misc proton patches and hacks - Notably contains fixes for some native vk games (such as Doom Eternal) as well as Rockstar launcher
 # Also enables some winevulkan performance optimizations - https://github.com/Joshua-Ashton/proton-wine/tree/winevulkan-opt (fs hack) - https://github.com/Joshua-Ashton/wine/commits/winevulkan-opt-mainline (no fs hack)
-_protonify="false"
+_protonify="true"
 
 
 #### USER PATCHES - See README in ./wine-tkg-userpatches dir for instructions ####
 
 # community patches - add patches (separated by a space) of your choice by name from the community-patches dir - https://github.com/Frogging-Family/community-patches
 # example: _community_patches="amdags.mypatch GNUTLShack.mypatch"
-_community_patches=""
+_community_patches="0002-proton_LFH-staging.mypatch roblox_mouse_fix.mypatch"
 
 # Automatically update community-patches from git - set to false if you don't want this behaviour, useful for example if you have your own fork.
 _community_patches_auto_update="true"
diff --git a/wine-tkg-git/wine-tkg-profiles/advanced-customization.cfg b/wine-tkg-git/wine-tkg-profiles/advanced-customization.cfg
index 1391778..ca703ec 100644
--- a/wine-tkg-git/wine-tkg-profiles/advanced-customization.cfg
+++ b/wine-tkg-git/wine-tkg-profiles/advanced-customization.cfg
@@ -270,7 +270,7 @@ _fsync_futex2="true"
 _user_patches="true"
 
 # Apply all user patches without confirmation - !!! NOT RECOMMENDED !!!
-_user_patches_no_confirm="false"
+_user_patches_no_confirm="true"
 
 # Set to "false" to prompt about all non-critical hotfix patches at build time, or "ignore" to ignore all non-critical hotfix patches without confirmation
 # Default ("true") will apply all non-critical hotfix patches without confirmation
diff --git a/wine-tkg-git/wine-tkg-scripts/prepare.sh b/wine-tkg-git/wine-tkg-scripts/prepare.sh
index 8567540..4198821 100644
--- a/wine-tkg-git/wine-tkg-scripts/prepare.sh
+++ b/wine-tkg-git/wine-tkg-scripts/prepare.sh
@@ -789,6 +789,10 @@ _prepare() {
 
 	if [ "$_unfrog" != "true" ]; then
 	  source "$_where"/wine-tkg-patches/reverts
+		if ! git merge-base --is-ancestor 197f2ee6c0e2dd70cfac6e15ede7fc0e38948188 HEAD; then
+			_committorevert=db2b266c57b73e1a16785213ce923b749c84400e && nonuser_reverter
+			echo -e "( Roblox Studio black mouse unbreak reverts applied )\n" >> "$_where"/last_build_config.log
+		fi
 	fi
 
 	_commitmsg="01-reverts" _committer
