diff --git a/wine-tkg-git/customization.cfg b/wine-tkg-git/customization.cfg
index 6c6fa34..fbbb795 100644
--- a/wine-tkg-git/customization.cfg
+++ b/wine-tkg-git/customization.cfg
@@ -96,7 +96,7 @@ _assettocorsa_hudperf_fix="false"
 
 # Fixes for Mortal Kombat 11 - Requires staging, _proton_fs_hack="true", native mfplat (win7) or staging mfplat support and a different GPU driver than RADV
 # On Wine 5.2 (up to b1c748c8) and lower, it needs to be toogled on with the WINE_LOW_USER_SPACE_LIMIT=1 envvar
-_mk11_fix="true"
+_mk11_fix="false"
 
 # Workaround for Final Fantasy XIV Launcher 404 error - Thanks @varris1 ! - Fixed by d535df42f665a097ec721b10fb49d7b18f899be9 (4.10)
 # Found to also enable the new launcher (that came with the 5.1 update) to work *with issues*
@@ -149,7 +149,7 @@ _protonify="false"
 
 # community patches - add patches (separated by a space) of your choice by name from the community-patches dir - https://github.com/Frogging-Family/community-patches
 # example: _community_patches="amdags.mypatch GNUTLShack.mypatch"
-_community_patches=""
+_community_patches="roblox_mouse_fix.mypatch"
 
 # Automatically update community-patches from git - set to false if you don't want this behaviour, useful for example if you have your own fork.
 _community_patches_auto_update="true"
diff --git a/wine-tkg-git/wine-tkg-profiles/advanced-customization.cfg b/wine-tkg-git/wine-tkg-profiles/advanced-customization.cfg
index 61ccce4..99bcd4c 100644
--- a/wine-tkg-git/wine-tkg-profiles/advanced-customization.cfg
+++ b/wine-tkg-git/wine-tkg-profiles/advanced-customization.cfg
@@ -128,9 +128,9 @@ _localbuild=""
 _plain_mirrorsrc="true"
 
 # Sets custom configure-args for 64-bit, separated by a space (example: "--without-mingw --with-vkd3d")
-_configure_userargs64="--with-x --with-gstreamer --with-xattr"
+_configure_userargs64="--with-x --with-gstreamer --with-xattr --without-xpresent"
 # Sets custom configure-args for 32-bit, separated by a space (example: "--without-mingw --with-vkd3d")
-_configure_userargs32="--with-x --with-gstreamer --with-xattr"
+_configure_userargs32="--with-x --with-gstreamer --with-xattr --without-xpresent"
 
 
 # esync version - https://github.com/zfigura/wine/releases - If you want to use a specific esync version. For 3.20 or higher, use "ce79346". For 3.19 to 3.17, use "b4478b7". For 3.16 and lower, use "5898a69".
@@ -269,11 +269,11 @@ _fsync_futex2="true"
 _user_patches="true"
 
 # Apply all user patches without confirmation - !!! NOT RECOMMENDED !!!
-_user_patches_no_confirm="false"
+_user_patches_no_confirm="true"
 
 # Set to "true" to apply all hotfix patches without confirmation, to "ignore" to ignore all hotfix patches without confirmation
 # Default ("false") will prompt at build time
-_hotfixes_no_confirm="false"
+_hotfixes_no_confirm="ignore"
 # Set to false to disable staging mfplat restoration in case a hotfix is available and _hotfixes_no_confirm is set to "true"
 _hotfixansw_staging_mfplat=""
 # Set to false to disable staging pulseaudio restoration in case a hotfix is available and _hotfixes_no_confirm is set to "true"
diff --git a/wine-tkg-git/wine-tkg-scripts/prepare.sh b/wine-tkg-git/wine-tkg-scripts/prepare.sh
index 2b3d191..6b47bce 100644
--- a/wine-tkg-git/wine-tkg-scripts/prepare.sh
+++ b/wine-tkg-git/wine-tkg-scripts/prepare.sh
@@ -683,6 +683,10 @@ _prepare() {

 	if [ "$_unfrog" != "true" ]; then
 	  source "$_where"/wine-tkg-patches/reverts
+		if ! git merge-base --is-ancestor 197f2ee6c0e2dd70cfac6e15ede7fc0e38948188 HEAD; then
+			_committorevert=db2b266c57b73e1a16785213ce923b749c84400e && nonuser_reverter
+			echo -e "( Roblox Studio black mouse unbreak reverts applied )\n" >> "$_where"/last_build_config.log
+		fi
 	fi

 	_commitmsg="01-reverts" _committer
